monai/utils/enums.py

```diff
@@ -692,3 +703,39 @@ class AlgoKeys(StrEnum):
     ALGO = "algo_instance"
     IS_TRAINED = "is_trained"
     SCORE = "best_metric"
+
+
+class AdversarialKeys(StrEnum):
```

Comment:
Would you mind adding a brief description here and also for `AdversarialIterationEvents`?